docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -24,31 +24,52 @@ To create custom SQL query Expectations, you'll need to use the GX API. See [Cus
+| Data Integrity     | `expect_column_pair_values_to_be_equal`                   | Expect the values in column A to be the same as column B.                                                                              |
+| Data Integrity     | `expect_multicolumn_sum_to_equal`                         | Expect that the sum of row values in a specified column list is the same for each row, and equal to a specified sum total.             |
+| Distribution       | `expect_column_pair_values_A_to_be_greater_than_B`        | Expect the values in column A to be greater than column B.                                                                             |
+| Distribution       | `expect_column_values_to_be_between`                      | Expect the column entries to be between a minimum value and a maximum value (inclusive).                                               |
+| Distribution       | `expect_column_z_scores_to_be_less_than`                  | Expect the Z-scores of a column's values to be less than a given threshold.                                                            |
+| Distribution       | `expect_column_stdev_to_be_between`                       | Expect the column standard deviation to be between a minimum value and a maximum value.                                                |
+| Distribution       | `expect_column_sum_to_be_between`                         | Expect the column to sum to be between a minimum value and a maximum value.                                                            |
+| Missingness        | `expect_column_values_to_be_null`                         | Expect the column values to be null.                                                                                                   |
+| Missingness        | `expect_column_values_to_not_be_null`                     | Expect the column values to not be null.                                                                                               |
+| Numerical Data     | `expect_column_max_to_be_between`                         | Expect the column maximum to be between a minimum and a maximum value.                                                                 |
+| Numerical Data     | `expect_column_mean_to_be_between`                        | Expect the column mean to be between a minimum and a maximum value (inclusive).                                                        |
+| Numerical Data     | `expect_column_median_to_be_between`                      | Expect the column median to be between a minimum and a maximum value.                                                                  |
+| Numerical Data     | `expect_column_min_to_be_between`                         | Expect the column minimum to be between a minimum value and a maximum value.                                                           |
```

Comment:
Again -- is the median/minimum between a minimum value and a maximum value inclusive of the minimum and maximum values?  Or is it strictly greater than the minimum but less than the maximum?